mmv1/products/gkeonprem/BareMetalAdminCluster.yaml

```diff
@@ -30,15 +30,15 @@ examples:
     min_version: beta
     primary_resource_id: "admin-cluster-basic"
     vars:
-      name: "basic"
+      name: "my-cluster"
```

Comment:
Having dashes in these values triggers addition of `tf-test-` at the start of the value in the generated test - [see here](https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-8306-old..auto-pr-8306#diff-6e3978acf03b97ec7fbf9b27650af9d594402a7fc2ca690eb8c56dd298e74d56L62-L63)